cloudify/workflows/local.py

```diff
@@ -89,7 +89,8 @@ def outputs(self):
             get_node_instances_method=self.storage.get_node_instances,
             get_node_instance_method=self.storage.get_node_instance,
             get_node_method=self.storage.get_node,
-            get_secret_method=self.storage.get_secret)
+            get_secret_method=self.storage.get_secret,
+            get_capability_method=self.storage.get_capability)
```

Comment:
not implemented yet, right?

Reply:
It is, in https://github.com/cloudify-cosmo/cloudify-manager/pull/1490